recipe/meta.yaml

```
@@ -12,47 +12,40 @@ source:
   sha256: {{ sha256sum }}
 
 build:
-  number: 3
-  skip: True  # [win and py27]
-  detect_binary_files_with_prefix: False
+  number: 0
+  skip: True  # [win][py36]
```

I'm not sure that syntax works? Did you mean:
```suggestion
  skip: True  # [win or py36]
```

---

Yes, that is what I meant. Thank you. Fixed it.